README.md

```diff
@@ -155,15 +155,26 @@ Container can be built via running `make docker-build`.
 
 ## Commitment Schemas
 
-Commitments returned from the EigenDA Proxy adhere to the following byte encoding:
+### Optimism Commitment Mode
+For `alt-da` clients running on Optimism, the following commitment schema is supported:
 
 ```
- 0        1        2         3               N
- |--------|--------|---------|---------------|
-  commit   da layer  version  raw commitment
+ 0        1        2         3                N
+ |--------|--------|---------|----------------|
+  commit   da layer  version   raw commitment
```

Comment:
<img width="441" alt="image" src="https://github.com/user-attachments/assets/56fecf3c-0f2a-4150-896d-3ce6ff018b84">

So how come the version byte doesn't appear in the specs? https://specs.optimism.io/experimental/alt-da.html#input-commitment-submission

Reply:
this is unique to certificate versioning so we can target different verification paths in the future